packages/paste-website/src/pages/patterns/empty-state/index.mdx

```diff
@@ -210,109 +192,193 @@ Open the sandbox in a new tab and resize the browser for an example of responsiv
+        <Td scope="row">A user cleared all the data (like alerts or notifications)</Td>
+        <Td><b>No new [things]</b> <br/> To review previous [things], go to [page].</Td>
+        <Td>Optional</Td>
       </Tr>
       <Tr verticalAlign="top">
-        <Th scope="row">Change regional data location</Th>
-        <Td>The customer needs to swap regions to access the data.</Td>
-        <Td>"View event logs in the United States region"</Td>
+        <Td scope="row">A preview is populated after the user inputs info elsewhere on the page</Td>
+        <Td><b>No preview yet</b> <br/> To see what your [thing] will look like, [do a task].</Td>
+        <Td>Optional</Td>
+      </Tr>
+    </TBody>
+  </Table>
+</Box>
+
+<Box marginBottom="space70">
+  <iframe
+    src="https://codesandbox.io/embed/empty-state-requiring-a-customer-to-wait-uyxu3?fontsize=14&hidenavigation=1&theme=dark&view=preview"
+    style={{width: '100%', height: '350px', border: '0', borderRadius: '4px', overflow: 'hidden'}}
+    title="Empty state requiring a customer to wait"
+    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
+    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
+  ></iframe>
+</Box>
+
+### Error states
+
+Error empty states can result from problems like a system issue or access issue. The content should explain what went wrong and tell the user how to fix the issue. Consider including an error icon to complement the content. If the error empty state occurs in a small component (a card, for example), prioritize communicating how to fix the problem. For error message guidance, review the [error state pattern] (patterns/error-state).
+
+#### Examples
+
+<Box marginBottom="space70">
+  <Table scrollHorizontally>
+    <THead>
+      <Tr>
+        <Th width="200px">Use case</Th>
+        <Th>Sample content format</Th>
+        <Th width="200px">CTA</Th>
+      </Tr>
+    </THead>
+    <TBody>
+      <Tr verticalAlign="top">
+        <Td scope="row">Something went wrong with the system</Td>
         <Td>
-          <ChangeRegionImage />
-          <Anchor href="https://library.twilio.com/collections/view/963D9AB3-47A0-4C13-978BC97108C1EBCC/?mediaId=F44538E4-FD2E-45B2-806402557124153C">
-            Download from the Brand library
-          </Anchor>
+          <b>We could not load [thing]</b> <br/> Try reloading the page.
         </Td>
+        <Td>Optional</Td>
+      </Tr>
+      <Tr verticalAlign="top">
+        <Td scope="row">User doesn’t have access or permissions</Td>
+        <Td><b>No [thing] yet</b> <br/> You'll see [thing] here when [something happens].</Td>
+        <Td>Optional</Td>
+      </Tr>
+   <Tr verticalAlign="top">
+        <Td scope="row">404 page</Td>
+        <Td><b>Page not found</b> <br/> he page you're looking for either doesn't exist or was moved.</Td>
+        <Td>Go back</Td>
```

Comment:
thought (non-blocking): The CTA column header is a little confusing to me. In the previous table the entire column was marked as "Optional", which made me thing the CTA column header should be renamed to "CTA required?". But in this table it jumps from `Optional` to `Go back`, which makes me do a double-take at whether the CTA is supposed to say "Optional".  

Maybe a suggestion instead is to provide an example CTA but append it with `(optional)`.  Ex:
```
<Td><b>No [thing] yet</b> <br/> You'll see [thing] here when [something happens].</Td>
<Td>Return to the [thing] page (optional)</Td>
```

Reply:
+1